src/views/storagemigrations/list/hooks/useStorageMigrationResources.ts

```diff
@@ -28,6 +28,8 @@ const useStorageMigrationResources: UseStorageMigrationResources = () => {
     namespaced: true,
   });
 
+  const storageMigPlan = migPlans.filter((plan) => plan?.spec?.persistentVolumes?.length > 0);
```

Comment:
1. the code needs to handle load-in-progress and error states - I would go with extra guards i.e. `migPlans?.filter(...) ?? []`
2. it's a list but the name suggests single entity

Reply:
solved